externs/aria.js

```diff
@@ -16,3 +16,9 @@ Element.prototype.ariaLabel;
 
 /** @type {string} */
 Element.prototype.ariaPressed;
+
+/** @type {string} */
+Element.prototype.ariaSelected;
```

Comment:
We also set the `aria-label` attribute. That could be replaced also.
Using the `ARIAMixin` version would remove the need for us to have the `shaka.ui.Constants.ARIA_LABEL` constant, which is nice.

Reply:
Done!